server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateIndexService.java

```diff
@@ -395,7 +395,8 @@ private ClusterState applyCreateIndexWithTemporaryService(final ClusterState cur
             try {
                 updateIndexMappingsAndBuildSortOrder(indexService, request, mappings, sourceMetadata);
             } catch (Exception e) {
-                logger.debug("failed on parsing mappings on index creation [{}]", request.index());
+                logger.log(silent ? Level.DEBUG : Level.INFO,
```

Comment:
Can we also include the stacktrace of the error?

Reply:
I put that in initially and then changed my mind, as I have an aversion to logging stack traces from user errors.  It would make debugging easier though, I agree, so I'll add it back.